tests/roles/mariadb_copy/templates/post_checks.bash

```diff
@@ -8,19 +8,19 @@
 {% endif %}
 
 # use 'oc exec' and 'mysql -rs' to maintain formatting
-dbs=$(oc exec openstack-galera-0 -c galera -- mysql -rs -uroot -p"$PODIFIED_DB_ROOT_PASSWORD" -e 'SHOW databases;')
-echo $dbs | grep -Eq '\bkeystone\b'
+dbs=$(oc exec openstack-galera-0 -c galera -- mysql -rs -uroot "-p$PODIFIED_DB_ROOT_PASSWORD" -e 'SHOW databases;')
```

Comment:
"-p$PODIFIED_DB_ROOT_PASSWORD" ?

Reply:
-p"$PODIFIED_DB_ROOT_PASSWORD"

Reply:
There is no difference between these two approaches in Bash.

Reply:
The reason this is edited in this patch is that i re-copy-pasted what was in docs into the tests, to make them 100% in sync again. Both of the approaches that you guys quoted work the same, they're both correct AFAIK.

Reply:
Actually the `"-p$PODIFIED_DB_ROOT_PASSWORD"` variant is probably the best because `-p"$PODIFIED_DB_ROOT_PASSWORD"` might make the admin tempted to put a space after the -p like `-p "$PODIFIED_DB_ROOT_PASSWORD"` and i think that would break. IIRC the mariadb client requires the password to be included in the same command line argument as the `-p` flag, not split into two by a space.

Reply:
ack, thank you